docs/admin/backup.rst

```diff
@@ -0,0 +1,177 @@
+Backup and Restore
+==================
+
+.. include:: ../includes/top-warning.rst
+
+QubesOS has a `backup utility <https://www.qubes-os.org/doc/backup-restore/>`_ 
+that allows for backup and restoration of user-specified VMs. 
+
+To perform backups, you will need:
+
+ - an encrypted storage medium
+ - a secure place to store backup credentials    
+
+Backup 
+------
+
+Prepare files in ``dom0``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Files outside the ``dom0`` home folder aren't normally backed up by the 
+Qubes backup tool. Preserve key configuration files by coping them to the 
+home directory. 
+
+In a ``dom0`` Terminal via **Q ▸ Terminal Emulator**:
+
+  .. code-block:: sh
+
+    mkdir ~/backup/
+    cp -r /usr/share/securedrop-workstation-dom0-config/ ~/backup/
+
+If you have customized :doc:`clipboard access <managing_clipboard>` 
+to SecureDrop Workstation, or have made any other customizations 
+involving RPC policies, back up related configuration files:   
+
+  .. code-block:: sh
+
+   mkdir -p ~/backup/etc/qubes
+   mkdir ~/backup/etc/qubes-rpc
+   cp -a /etc/qubes/* ~/backup/etc/qubes/
+   cp -a /etc/qubes-rpc/* ~/backup/etc/qubes-rpc
+
+Back up SecureDrop Workstation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure your storage medium is plugged in, attached to a VM, and mounted.
+(LUKS-encrypted storage media may be attached to ``sd-usb``, or you may use a custom 
+VM of your choosing, if you are relying on another form of encryption such as 
+Veracrypt).
+
+Navigate to **Q ▸ System Tools ▸ Backup Qubes**, and move all VMs from 
+"Selected" to "Available" by pressing the ``<<`` button. 
+
+To target a VM for backup, highlight it and move it into the "Selected" 
+column by pressing the ``>`` button. Select the following VMs:
+
+ - ``dom0``
+ - ``vault``
+ - any customized VMs that you wish to preserve.
+
+You do not need to back up the ``sd-`` VMs.
+
+Click "Next", and in "Backup destination," specify the VM and directory 
+corresponding to your storage medium's current mount point.
```

Comment:
I've got light concerns about how little detail we're providing on the mechanics of creating a LUKS-encrypted volume, mounting it, and unlocking it. My personal backup procedures are roughly:

1. create `backup` appvm with no network
2. connect external hard drive , use sys-usb to connect it to `backup`
3. run cli `crytsetup luksOpen` and `mount` commands (bleh)
4. then switch back to dom0 and use the backup utility

I don't think we need to go into that much detail in our docs, it's much better to link out to upstream docs, as you've done above. It'd be an interesting exercise to ask someone without a ton of Qubes familiarity to run through these backup docs. Maybe we can try that at a later stage of review. =)

Reply:
I would expect a LUKS-encrypted drive to be automatically attached to `sd-devices` on fully provisioned Workstation. If that's the case, fine to use that to perform the backup!

Reply:
Ah, right: above it was referred to as `sd-usb`, and I'd planned to re-target that as `sys-usb`. Looks like `sd-devices` is what we want, and it's already addressed in a comment above.

Reply:
confirmed this works as expected and documented